scripts/orca-singularity

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
I suggest adding `set -eu` right after the shebang line. I do that for every shell script. `-e` means exit immediately after an error. `-u` means that using an undefined environment variable is an error.